util/cron/nightlysubs.pm

```diff
@@ -117,7 +117,11 @@ sub endMailHeader {
 }
 
 sub endMailChplenv {
-    my $chplenv = `$chplhomedir/util/printchplenv --debug`;
+    my $ch = $chplhomedir;
```

Comment:
Your logic makes sense to me, but the name chplhomedir seems to imply that it is using CHPL_HOME.  If that's not what it's doing in most cases, then perhaps the variable is poorly named.  And since there are cases where the relative path is needed, and cases where CHPL_HOME is needed, maybe those use cases should be using separate variables instead of overloading.  I think that's outside the scope of this patch, though.